kubernetes/annotations_and_labels.md

```diff
@@ -161,6 +179,20 @@ This page defines common annotations and labels we set in Kubernetes objects.
 
 #### Tenant Cluster
 
+### App Labels
```

Comment:
Someone decided to skip one level in the following paragraphs so I think you want to do `##`.

```suggestion
## Chart Templates Labels
```

Reply:
Do you mean that all the sub-headings of _"Common Labels"_ should be one level up? E.g. that `### Labels Set In Custom Resources` should actually be `## Labels Set In Custom Resources`?

Reply:
Sorry my mind was in a dark place apparently. `###` is good.

Reply:
Updated my suggestion.

Reply:
So how about something more explicit like _Labels Set On Objects Installed By Charts_?

Reply:
Went with that, let me know if that works.